lambdas/submit-answer/tests/submit-answer-handler.test.ts

```diff
@@ -2,9 +2,122 @@ import { SubmitAnswerHandler } from "../src/submit-answer-handler";
 import { Context } from "aws-lambda";
 
 describe("submit-answer-handler", () => {
-  it("should print Hello, World!", async () => {
-    const submitAnswerHandler = new SubmitAnswerHandler();
-    const result = await submitAnswerHandler.handler({}, {} as Context);
-    expect(result).toStrictEqual("Hello, World!");
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetAllMocks();
+  });
+
+  describe("happy path scenarios", () => {
+    it("it should return the response from the HMRC API", async () => {
+      const hmrcResponse = {};
```

Comment:
Is this what we get back from the response `{}`

Reply:
I think we should look into converting some of these lambda into Api integration, I feel this can be done the statemachine. We can then check the response if it json or if it is text and do something with it.